src/main/scala/FbEvents.scala

```diff
@@ -6,17 +6,28 @@ import scalaj.http._
 
 object FbEvents {
   case class FbEvent(id: String, name: String, description: Option[String], startTime: ZonedDateTime, endTime: Option[ZonedDateTime]);
-  case class Response(data: List[FbEvent]);
+  case class Cursors(before: String, after: String);
+  case class Paging(cursors: Cursors);
```

Comment:
Wouldn't it be simpler to use the `next` field instead of the `Cursors`? (https://developers.facebook.com/docs/graph-api/using-graph-api)